frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/cloud-init-tab/cloud-init-tab.tsx

```diff
@@ -84,7 +84,7 @@ const CloudInitFormRows: React.FC<CloudInitFormRowsProps> = ({
     authKeys.push('');
   }
   return (
-    <>
+    <span className="co-m-pane__body co-m-pane__form">
```

Comment:
not sure we should be adding additional elements, especially span for the whole list.

Can we add this to `<Form>` element conditionally depending on `isForm` variable?

Reply:
hmm, trying, sec ...

Reply:
done, much nicer :+1: